bokeh/models/plots.py

```diff
@@ -692,3 +692,10 @@ def webgl(self, webgl):
         When set to ``webgl``, glyphs without a WebGL rendering implementation
         will fall back to rendering onto 2D canvas.
     """)
+
+    aspect_ratio = Either(Bool, Float, default=False, help="""
+    Specify the aspect ratio behavior of the plot. Default is ``False`` which
+    means that the glyph will fill the whole plot area. ``True`` indicates that
+    the glyphs should maintain the an aspect ratio of 1. A ``Float``value can
```

Comment:
lack of space after last backtick is what is causing the one test failure (sphinx barfs)